app/build.gradle

```diff
@@ -31,8 +31,9 @@ dependencies {
 	implementation project(":sdk")
 
 	// google
-	implementation 'com.google.android.gms:play-services-location:11.4.0'
-	implementation 'com.google.android.gms:play-services-maps:11.4.0'
+	implementation 'com.google.android.gms:play-services-location:11.4.2'
+	implementation 'com.google.android.gms:play-services-maps:11.4.2'
+	implementation 'com.google.android.gms:play-services-auth:11.4.2'
```

Comment:
the commits for demo and sdk should be split!

Reply:
:heavy_check_mark: